src/app/page.tsx

```diff
@@ -1,11 +1,14 @@
 'use client';
 
+import { initJsStore } from '@/api/database/indexeddb/connection';
 import { AlertProvider } from '@/context/AlertContext';
 import { Alert } from '@/components/Alert';
 import { HistoricalView } from '@/components/HistoricalView';
 import { CreateButton } from '@/components/CreateButton';
 
 const Home = (): JSX.Element => {
+  initJsStore();
```

Comment:
this will be run multiple times on every re-render, please use `useEffect()`

Reply:
updated in https://github.com/RR-Ardhika/murojaah/pull/8/commits/cc17e6ff14033e2024085420713e9d47a63ac800, thank you